<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_dialog</class>
 <widget class="QDialog" name="login_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="browse_cert_button">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Key:</string>
       </property>
       <property name="buddy">
        <cstring>key_path</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="key_path"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="browse_key_button">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Certificate:</string>
       </property>
       <property name="buddy">
        <cstring>cert_path</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="cert_path">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="remember_me">
       <property name="text">
        <string>Remember me</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="login_automatically">
       <property name="text">
        <string>Login automatically</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="register_button">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="login_button">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cert_path</tabstop>
  <tabstop>browse_cert_button</tabstop>
  <tabstop>key_path</tabstop>
  <tabstop>browse_key_button</tabstop>
  <tabstop>remember_me</tabstop>
  <tabstop>login_automatically</tabstop>
  <tabstop>register_button</tabstop>
  <tabstop>login_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
